docs/standard/class-libraries.md

```diff
@@ -42,7 +42,7 @@ Many library developers have switched from producing multiple platform-specific
 
```

Comment:
The heading should read ".NET Standard class libraries", not ".NET Core Class Libraries"